src/app/forms.py

```diff
@@ -80,6 +80,7 @@ def __init__(self, *args, **kwargs):
     shortIdentifier = forms.CharField(label='Short identifier', max_length=25)
     sourceUrl = forms.CharField(label='Source / URL', required=False)
     osiApproved = forms.CharField(label="OSI Status", widget=forms.Select(choices=OSI_CHOICES))
+    examples = forms.CharField(label='Example Projects / URL', required=False)
```

Comment:
This will cause the same issue as https://github.com/spdx/spdx-online-tools/issues/249. Since its a Char Field, so form only accept one example. In views.py
`licenseExamples = [form.cleaned_data['examples']]` will return only one example. 

We either to need to change the forms.py to accept multiple urls or send the urls as comma separated/json formatted from javascript.

Will look into it and commit the changes along with https://github.com/spdx/spdx-online-tools/issues/249 fix.

Reply:
Agreed - any help appreciated.  I may make an attempt at a fix tomorrow based on your suggestions - I'll update the PR and issue if I'm successful.